public/v4/apps/serpbear.yml

```diff
@@ -0,0 +1,59 @@
+captainVersion: 4
+services:
+    $$cap_appname:
+        restart: always
+        image: 'towfiqi/serpbear:$$cap_serpbear_version'
+        environment:
+            NEXT_PUBLIC_BASE_URL: 'http://$$cap_appname.$$cap_root_domain'
```

Comment:
Where is `NEXT_PUBLIC_BASE_URL` being used?

Reply:
Ah I think you meant to use `NEXT_PUBLIC_APP_URL`

Reply:
Good catch, thanks for noticing